src/pages/event/[slug].js

```diff
@@ -34,10 +39,14 @@ export async function getStaticProps({ params }) {
     };
 
   const eventData = await getEventByName({ name: params.slug });
+  const allCollections = await getAllCollections();
+  const allPaintings = await getAllPaintings();
+  const combineResponsesInArray = [...allCollections, ...allPaintings];
 
   return {
     props: {
       eventData,
+      combineResponsesInArray,
```

Comment:
This variable name is important, I would change it to something like `paintingsAndCollections`.

Combined responses is quite generic, a response could be anything.